docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Configure_Azure_IoT_Central_integration.md

```diff
@@ -0,0 +1,35 @@
+# Connect integration
```

Comment:
In DPS we had `Configure integration`, but in fact `Connect integration` sounds better. I would then change it there so that headlines are consistent.